std_msgs/Header header              # ROS time assigned to RV when used e.g. in odometry 
geometry_msgs/Quaternion quaternion 
uint32 timestamp                    # MCU timestamp
